src/librustc_parse/lib.rs

```diff
@@ -277,7 +277,7 @@ pub fn parse_in_attr<'a, T>(
 ) -> PResult<'a, T> {
     let mut parser = Parser::new(
         sess,
-        attr.get_normal_item().tokens.clone(),
+        attr.get_normal_item().args.outer_tokens(),
```

Comment:
```suggestion
        // FIXME(#66940, Centril | petrochenkov): go through each usage of
        // this function and try to eliminate calls to it so we avoid re-parsing.
        attr.get_normal_item().args.outer_tokens(),
```

Reply:
Don't know when I'll have time for it, but it seems like something I could give a try. :)